src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2066,7 +2078,8 @@ export default class Aircraft {
                 }
 
                 this.target.speed = crange(3, offset[1], 10, this.model.speed.landing, this.fms.currentWaypoint().start_speed);
-            } else if ((this.altitude - runway_elevation) >= 300 && this.mode === FLIGHT_MODES.LANDING) {
+            }
```

Comment:
I was explicitly changing this to that. I consider it to be way less readable when I don't see an `else if` in vertical alignment with its own closing bracket. eg seeing "if" --> "{" --> "{" --> "{" when I look down a column seems to me to be less readable.

Again, I can change (I'm actually enjoying learning all this), just am curious as to the reasoning here.